src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSectionRenderer.cs

```diff
@@ -70,7 +70,7 @@ void IAppearanceObserver.OnAppearanceChanged(ShellAppearance appearance)
 		ShellSection _shellSection;
 		bool _ignorePopCall;
 
-		bool _popRequested;
+		internal bool popRequested;
```

Comment:
Can you change this to a full property instead of a field? 

I'd rather not mark fields anything but private